src/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -238,23 +244,32 @@ export const StartDate: React.FC<Props> = ({
         setDateValue(newDateValue)
         setShowError(false)
         if (newDateValue === null) {
-          removeStartDate({
-            variables: {
-              quoteId: offerId,
-            },
-          }).catch(handleFail)
+          Promise.all(
+            quoteIds.map((quoteId) =>
+              removeStartDate({
+                variables: {
+                  quoteId,
+                },
+              }),
+            ),
+          ).catch(handleFail)
         } else {
-          setStartDate({
-            variables: {
-              quoteId: offerId,
-              date: format(newDateValue, gqlDateFormat),
-            },
-          })
+          Promise.all(
+            quoteIds.map((quoteId) =>
+              setStartDate({
+                variables: {
+                  quoteId,
+                  date: format(newDateValue, gqlDateFormat),
+                },
+              }),
+            ),
+          )
+            .catch(handleFail)
```

Comment:
The design decision for now is to use the same start date for both insurances in the bundle.